### ISSUE TITLE
PostgreSQL separate folder for function/procedure/trigger function

### ISSUE THREAD

**Issue by bazmap** [dbeaver/dbeaver#22220]
In pgAdmin, there is two different folder for functions and trigger functions.
It could be great to have different folders for functions and trigger functions and the best ? for procedures too.
It could be easier to distinct each of these object in the database.

**Comment by LonwoLonwo** [comment]
Hello @bazmap 

Thanks for the idea. 

I'm unsure if we want to add more entities in our navigator tree for PostgreSQL. 
Maybe we could do it with a special setting or advanced navigator view representation.

**Comment by bazmap** [comment]
Yeah, no problem, it could be a good idea to have an advanced view.
Thanks for the review
